src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,85 @@
+
+Error sampling decisions are taken based on the event data.
+
+A sampling rule is selected from the available sampling rules by going through the list of dynamic sampling rules for
+errors and picking up the first matching rule that matches the event data. Rule matching is described in more details
+below.
+
+## Transactions
+
+Transactions can be considered either as independent events or in the context of all other transactions belonging to a
+trace (see TODO tracing link for more info on traces).
+
+Sentry offers the possibility to sample transactions either independently or as part of a trace; there are different
+benefits for each case.
+
+Sampling transactions using transaction traces is advantageous when trying to understand transactions in the context
+of the global system. In this case we are not interested in seeing a particular transaction, but the whole group of
+transactions in a trace so sampling should occur at the trace level.
+
+When sampling transaction traces the rules sampling rules are based on the transaction context, this means that the
+sampling decisions are based on the information extracted from the system that initiated the transaction. A
+transaction trace sample will rules are based on the following attributes (all attributes belong to the initial
+transaction, the transaction that started the trace):
+
+* project id
+* release
+* environment
+* user segment
+
+While sampling transaction traces gives a very good view of how transactions flow through the system, sometimes we are
+interested in analysing transactions from a particular service that happens not to be the initiator of transaction
+(e.g. a backend server). If we are interested in creating rules based on attributes specific to our service, for
+example we have a suspicion that a particular release might have some performance problems, we can sample based on
+transaction event.
+
+Individual transaction rules are based on the transaction attributes and not on the trace context attributes and
+therefore can target individual services that are not initiating traces.
+
+(TODO In the future we can say that these rules can use more attributes).
+
+# Relation with inbound filters
+
+(TODO Decide if we should keep this since we'll probably remove inbound filters in the future.)
+
+Inbound filters are a related but different concept. At a high level inbound filters may be viewed as a coarse sampling
+technique where only 0% and 100% sampling rates are available. The main difference between how filters and sampling
+rules operate is that filters compose while sampling rules do not, the example below will clarify how they operate.
+
+If we have two filters defined, one that removes all requests coming from Internet Explorer 9 or lower and one that
+filter requests coming from an application with release 1.* the resulting effect will be that all events that are
+coming either from Explorer 9 or that are coming from a client with release 1.* will be filtered out.
```

Comment:
Might we explain that this is similar to a join statement? 
```suggestion
With two filters defined - one that removes all requests coming from Internet Explorer 9 or lower, and one that
filters requests coming from an application with release 1.x - means that all events 
coming either from Explorer 9 or coming from a client with release 1.x will be filtered out.
```

Reply:
No, a join statement would be the wrong concept  (a join is a cartesian product of two sets). I think the easiest way to clarify what happens here is a logical "or" of two conditions. If we want to use sets to explain the concept the operation is a "union" (non discriminated). In this case, for me, thinking in terms of a logical "or" of two conditions is a bit more intuitive than thinking about sets (which would be the union in the space of all possible events of the subset of events that have release '1.*' and the subset of events that have the user agent "Explorer 9" :-) ).

Reply:
Regarding 1.x instead of 1.*  I'm probably biased but I think that 1.* would be better. The reason is that this is how we specify `glob` patterns, this is actually how a user would specify the release in the rules. So specifying '1.3.*' would match both release 1.3.1 and 1.3.7 but will **not** match 1.2.1. We are using `glob` matching in various places so most users will probably know what we are talking about. For now I'll leave it with '*' instead of 'x' but if you still feel that 'x' is clearer I'll change it.

Reply:
The "or" concept is better, I agree.

I don't have a strong opinion about using an asterisk rather than `1.x`, but asterisks are used for formatting bold (two asterisks) and italic (one asterisk) in Markdown, so you may end up with bolded or italicized statements where you don't expect them.